### ISSUE TITLE
Random Agent Spoofer and Virtualis service (online bank payment service)

### ISSUE THREAD

**Issue by Aliaspub** [dillbyrne/random-agent-spoofer#523]
Hi.
I can't connect to my personnal bank e-card payment service when Random Agent Spoofer is activated.
Here is the link to access to the service :
https://www.cmb.fr/domivirtualis/index.html#/

Config : Win10 64bits Home FF 51.0.1 stable release.

**Comment by BananaMangoFestival** [comment]
Hi.
Did you try to connect without `script injection`?

**Comment by Aliaspub** [comment]
I just tried. It still doesn't work even if if I disable this feature. Other idea ?

**Comment by BananaMangoFestival** [comment]
Maybe by disabling "Disable Authorization"?
You are using a random browser profile that changes periodically or one that never changes?
You could try to select only one firefox windows profile and see what happens

**Comment by Aliaspub** [comment]
This box was initially unticked. Even ticked, it doesn't work.

I am using a random profile (only for Desktop browsers) which change each time FF is restarted.
Even if I tick "Real profile", I can't connect to this service. I can only if I deactivate the add-on. So this is probably in "Header" or "Option" tab that the problem come from.

But if you don't have any other idea, please investigate for further version of the add-on. While, I deactivate it totally so that I can access to the bank service. Then I reactivate it.

An other question : Do you recommand to me a similar add-on for Opera at random ?
Thank you

**Comment by BananaMangoFestival** [comment]
Indeed, I'm quite sure that's in headers or options, could you show me your both tabs?
About your other question, I don't know any similar addon for opera...

**Comment by Aliaspub** [comment]
Here are screenshots from the Header and Option tabs :
http://zupimages.net/viewer.php?id=17/05/yr4h.png
http://zupimages.net/viewer.php?id=17/05/lpa4.png
http://zupimages.net/viewer.php?id=17/05/mlcz.png

**Comment by BananaMangoFestival** [comment]
You can check the same cases as me, your problem comes probably from your referer, sending nothing can be cause issues like connections to an account
https://image.noelshack.com/fichiers/2017/05/1485900187-2.png
https://image.noelshack.com/fichiers/2017/05/1485900222-1.png (there you have to type "referer" in about:config)

**Comment by Aliaspub** [comment]
I found the problem thanks to your screenshots.
I must disable ONLY the "Disable Referer" feature so that it works.
This is the only one website which this happens on. Your bank need to your referer. Isn't it an attempts to your privacy forcing you to indicate your referer like this ?

**Comment by BananaMangoFestival** [comment]
The problem was that you have completely disabled any http referer even when you move within a same website and you bank needs probably this referer to connecting you to your account
The best alternative which is proposed by [JonDonym](http://ip-check.info/), is to hide the referer only if you move to another website, but it remains the same within a website and changes when you switch from it
